CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- `defaultValues` prop.
+- Expose `useFormContext` through API.
```

Comment:
Why?

Reply:
I need access to the `control` object so I can pass it to `Controller`, which is used to wrap controlled components, such as the ones from Material.

Reply:
Can we expose control instead?

Reply:
Sure! But it needs to come from `useFormContext`. Do you have any suggestions?

Reply:
I'm not super familiar. Approving to unblock

Reply:
After re-reading the README, I've noticed that the form context can be accessed using the other hooks, so that change is not really necessary.